scanner/src/routes/+page.svelte

```diff
@@ -96,7 +104,7 @@
         if (contents.startsWith(IOXIO_TAGS_VERSION_PREFIX)) {
           const withoutVersion = contents.substring(IOXIO_TAGS_VERSION_PREFIX.length)
           const b45decoded = decodeBase45(withoutVersion)
-          const cborData = await parseCoseInsecure(b45decoded)
+          const cborData = await parseCoseInsecure(b45decoded.toString())
```

Comment:
The function takes in a `message: string`, and then does `stringToBuffer` to it. Instead of converting this `Buffer` instance `.toString()` and then back to `Buffer` again, how about just passing the `Buffer` in -> changing type to `message: Buffer`, and removing the `stringToBuffer` call?